FORMAT_ADIOS.md

```diff
@@ -0,0 +1,28 @@
+# Implementation Details: ADIOS1/2
```

Comment:
I would assume, one requirement that holds for all our backends is that names of datasets and groups should not clash? Should we mention that somewhere?

Reply:
Discussed on slack: no collision risk at the moment in the base standard.